HIPs/incubation/identus.md

```diff
@@ -0,0 +1,172 @@
+
+As part of the project evolution, in 2023, IOG decided to open-source Atala PRISM's codebase and contribute its main component — the Cloud Agent (previously known as the Open Enterprise Agent) — to [Hyperledger Labs](https://wiki.hyperledger.org/display/labs). In the open-source space, we've engaged with the community, opened communication channels, and hosted regular community meetings, leading to improvements based on user feedback. Our efforts have encouraged the development of several third-party PoCs and nurtured a growing community. While the project still has a few references to its roots as an internal project from IOG, it is already mostly decoupled, and the development is happening not only by developers from IOG but also by the community.
+
+We will increase our involvement in Hyperledger by adding additional decentralized identity components to the Identus project, enabling us to reach a wider audience and promote technology adoption.
+
+## Relationship to Hyperledger Aries
+
+Identus builds upon the foundational protocols defined in Hyperledger Aries RFCs, which provide secure communication and verifiable credential exchange standards.  While adhering to these Aries standards, Identus offers a distinct implementation using the Scala programming language, which aligns with other Aries-inspired frameworks such as AcaPy (Python) and AFJ (JavaScript).
+
+A key difference lies in Identus' emphasis on DIDComm v2, the latest version of the DIDComm messaging protocol, while many existing Aries implementations currently focus on DIDComm v1.
+
+## Collaboration Opportunities
+
+Identus and Aries hold the potential for valuable collaboration. Identus' experience with DIDComm v2 could aid in its adoption within Aries. Additionally, exploring the integration of Aries Askar as a secure storage and key management solution for Identus could prove beneficial.
+
+## Choosing Between Identus and Aries
+
+When selecting a framework for decentralized identity solutions, consider the following factors:
+- Programming Language Preference: Although writing business logic can be done in any language, if your development team favors Scala (JVM) based backends, Identus offers a natural fit. On the other hand, Aries provides implementations in Python and JavaScript, among others.
+- DIDComm Version: If your project's requirements necessitate DIDComm v2, Identus would be the more suitable choice.
+- Community: To determine whether Identus and Aries suit your project, you must evaluate the community composition. Although Identus aims to be VDR agnostic, it has a more substantial presence within the Cardano communities due to its association with IOG, which offers better synergies for use cases that target the Cardano ecosystem.
```

Comment:
What other VDRs does Identus support besides Cardano?

Reply:
At the moment, only Cardano. The plan is to improve the design to provide an interface so that the adoption of other VDRs can be done by implementing the interface